synfig-studio/ui/synfig-catalog.xml

```diff
@@ -2,14 +2,18 @@
 <glade-catalog name="synfigstudio" depends="gtk+" version="0.2" domain="synfigstudio">
```

Comment:
maybe bump the version?

Reply:
bumping to 0.3

Also, do I have to bump this from glade file?
```xml
...
<requires lib="synfigstudio" version="0.2"/>
...
```

Reply:
yes, it should. Sorry

Reply:
bumping it...